config/initializers/content_security_policy.rb

```diff
@@ -16,7 +16,11 @@
   #   # policy.report_uri "/csp-violation-report-endpoint"
 
   # allow webpack-dev-server host as allowed origin for connect-src
-  policy.connect_src :self, :https, "http://localhost:3035", "ws://localhost:3035" if Rails.env.development?
+  if Rails.env.development?
+    webpack_host = ENV.fetch("APP_DOMAIN") { "localhost" }
+    policy.connect_src :self, :https,
+                       "http://#{webpack_host}:3035", "ws://#{webpack_host}:3035"
```

Comment:
I don't think this logic is enough as `APP_DOMAIN` by default contains the port: 

https://github.com/thepracticaldev/dev.to/blob/4966783e18ab4b61c13bf32f084779b5b1da9dd1/Envfile#L11

Reply:
@rhymes Oh, TIL, thanks. Updated to accomodate.